main/getting-started/Documentation-Map.md

```diff
@@ -0,0 +1,69 @@
+# Agoric Documentation Map
+
+**tyg todo**: Add links to all these docs before publishing
+
+Agoric provides three main types of documentation, as well as links to
+"big picture" presentations and papers and links to our source code on GitHub. This page
+describes our documentation structure, where documents are, and when you should use them, 
+as well as what order to read them.
+
+Our three main documentation types are:
+- **Getting Started**: These are the first things you should
+read. They cover what to do to install, set up, and start
+using Agoric's smart contract tools. They also introduce those tools and
+their underlying concepts and designs.
+- **Guides**: These are more detailed and complete descriptions
+of our tools and their concepts, designs, and usage. They include
+descriptions and examples of how to do various tasks with Agoric
+software.
+- **Reference**: These are the specs for our APIs and commands,
+  as well as checklist tables that specify and explain the
+  steps for extended procedures, such as installing Agoric
+  software.
+
+Some documents fall into multiple categories. For example, our Agoric
+installation document is both a Reference and a Getting Started document.
+
+You should read them in this order:
+1. **Agoric Prerequisites**: What you need to do before installing
+   Agoric software and installing the Agoric SDK.
+2. **Introduction to ERTP**: A Getting Started document that
+  briefly explains the concepts and components of Agoric's Electronic
+  Rights Transfer Protocol (ERTP) which is the base of Agoric
```

Comment:
Maybe instead: "at the heart of Agoric software"? At the base implies the tech stack to me, but ERTP is near the top of the stock.